### DISCUSSION TITLE
how the hell did the error log get up to 282.6 GiB

### DISCUSSION THREAD

**Discussion by aquifolly** [JakeStanger/ironbar#451] [Q&A]
https://akko.wtf/notice/AeqeKmpXN9P9D6N54i

it was full of broken pipe errors

**Comment by JakeStanger** [comment]
Wow yep that's a problem. Mine ain't quite *that* bad but it's pretty out of hand:

```
λ ls -lah .local/share/ironbar/
Permissions Size User Date Modified Name
.rw-r--r--  619M jake 13 Feb 22:24   error.log
```

Currently there is no log rotation or size limit, so it'll grow indefinitely. That's definitely an issue, so I'll convert this to one and get it sorted.

**Reply by JakeStanger** [reply]
Tracking in #453

**Reply by JakeStanger** [reply]
Resolved, sorry about that!
